frontend/src/components/profile/profileSectionFooter/ProfileSectionFooter.tsx

```diff
@@ -0,0 +1,34 @@
+import { Typography } from '@mui/material';
+
+type color = 'gray' | 'red';
+
+type ProfileSectionFooterProps = {
```

Comment:
Since the `Delete account` section requires a red background color and a red border, I would either advise you to add a border prop.